src/client/auth/auth_err_test.go

```diff
@@ -1,16 +1,16 @@
 package auth
 
 import (
-	"fmt"
 	"testing"
 
+	"github.com/pachyderm/pachyderm/src/client/pkg/errors"
 	"github.com/pachyderm/pachyderm/src/client/pkg/require"
 )
 
 // grpcify returns an error e such that e.Error() is similar to what grpc
 // errors emit (though this doesn't convert 'error' to an actual GRPC error)
 func grpcify(err error) error {
-	return fmt.Errorf("rpc error: code = Unknown desc = %s", err.Error())
+	return errors.Errorf("rpc error: code = Unknown desc = %s", err.Error())
```

Comment:
Yeah, we can implement our own `Wrap` method if we want that lets us free format the error string. That error type would just have to implement the `Unwrap` function and then it'd satisfy our interfaces. I didn't seem worth it here since it's just a test, and `grpcify` isn't called on errors that came from elsewhere, so you're going to get essential the same stack you would have gotten. But if we run into this and care enough to work around it, it's very solvable.